src/main/java/de/adesso/anki/battle/transfer/BodyDTO.java

```diff
@@ -2,20 +2,67 @@
 
 import de.adesso.anki.battle.util.Position;
 import de.adesso.anki.battle.world.Body;
+import de.adesso.anki.battle.world.DynamicBody;
+import de.adesso.anki.battle.world.bodies.Vehicle;
+import de.adesso.anki.battle.world.bodies.roadpieces.Roadpiece;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'de.adesso.anki.battle.world.bodies.roadpieces.Roadpiece'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=32&issues=AWSshItHsk71k7lwCc1g&open=AWSshItHsk71k7lwCc1g)

[ISSUE_KEY]: AWSshItHsk71k7lwCc1g